docs/docs/overview/quick-start.md

```diff
@@ -63,7 +63,7 @@ const passwordResetTemplate = await notifire.registerTemplate({
 
 ### Trigger
 
-On the relevant business logic unit, just declare the trigger event, pass pre-defined parameters, you're done 🎊.
+On the relevant business logic unit, just declare the trigger event, pass pre-defined parameters, you'r done 🎊.
```

Comment:
Hey @REDMI7AMM I am unfamiliar with you'r , what does that mean? As far as I know you're is the correct usage here.
waiting to hear your thoughts